gen/dcos-config.yaml

```diff
@@ -2130,7 +2130,7 @@ package:
 {% endswitch %}
       [agent]
         ## Default data collection interval for all inputs
-        interval = "20s"
+        interval = "10s"
```

Comment:
This value is crucial knowledge to reliably report metrics from short-lived jobs (they need to live at least `interval` to make sure that their metrics enter the system). https://jira.mesosphere.com/browse/DCOS-49035

Reply:
👍 We probably ought to fix the dcos_statsd input so that it doesn't discard a container's metrics before they've been gathered.